### DISCUSSION TITLE
How to calculate Cost and total tokens on each request?

### DISCUSSION THREAD

**Discussion by varunahuja555** [langflow-ai/langflow#513] [General]
How to to calculate the cost and token on each request, for both chat webscoket/predict api?

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
